tools/create_index/index_generator.js

```diff
@@ -0,0 +1,50 @@
+var fs = require('fs')
+
+// We will fill in this object with our data
+var vuln = {}
+
+const createIndex = function (vulnDirectoryPath, writeDirectoryPath, filename) {
+  const files = fs.readdirSync(vulnDirectoryPath)
+  getVulnDirectoryContents(files, vulnDirectoryPath)
+  writeIndex(vuln, writeDirectoryPath, filename)
+}
+
+
+const getVulnDirectoryContents = function (entries, vulnDir) {
+  // if the first entry's value contains `'.json'` we can assume that all entries are files. Otherwise they're directories.
+  const entriesAreFiles = entries[0].includes('.json')
```

Comment:
It would be safer to use `stats.isDirectory()` here

Reply:
Done.